content/applications/productivity/iot/config/flash_sdcard.rst

```diff
@@ -1,37 +1,41 @@
-=====================
-Flashing your SD Card
-=====================
+====================
+Flashing the SD Card
+====================
 
-In some case, you may need to reflash your IoT Box’s SD Card to benefit
-from our latest updates.
+In some circumstances, the IoT Box’s SD Card may need to be reflashed to benefit from Odoo's latest
+IoT image update.
 
 Upgrade from the IoT Box homepage
 =================================
 
-Go to the IoT Box homepage, click on *Update*, next to the version number.
-If a new version of the IoT Box image is available, you will see a
-*Upgrade to ___* button a the bottom of the page, the IoT Box will then
-flash itself to the new version of the IoT Box. All of your configurations
-will be saved.
+Go to the IoT Box homepage, click on :guilabel:`Update`, next to the version number. If a new
+version of the IoT Box image is available, a :guilabel:`Upgrade to ___` button will appear at the
+bottom of the page, the IoT Box will then flash itself to the new version of the IoT Box. All of the
+previous configurations will be saved.
 
 .. note::
-   This process can take more than 30 minutes. Do not turn off or unplug
-   the IoT Box as it would leave it in an inconsistent state.
+   This process can take more than 30 minutes. Do not turn off or unplug the IoT Box as it would
+   leave it in an inconsistent state.
 
 .. image:: flash_sdcard/flash_sdcard01.png
    :align: center
+   :alt: IoT Box software upgrade in the IoT box console.
 
 Upgrade with Etcher
 ===================
 
-Go to Balena’s website and download `Etcher <https://www.balena.io/>`__.
-It’s a free and open-source utility used for burning image files.
-Install and launch it. Download the latest image from `nightly
-<http://nightly.odoo.com/master/iotbox/>`__.
+Go to Balena's website and download `Etcher <https://www.balena.io/>`__. It's a free and open-source
```

Comment:
Might be worth to add a paragraph before to explain that they require to do the operation with a computer on which they can connect the SD card (with a dongle if necessary).

Reply:
Excellent suggestion, I have added this in.